docs/design/coreclr/botr/crossgen2-compilation-structure-enhancements.md

```diff
@@ -0,0 +1,99 @@
+# Crossgen2 Driven Compilation structure enhancements
```

Comment:
Would `docs\design\features` be a better directory for this? Feels like BOTR has different kinds of documents.

Reply:
Sounds good to me. I missed that directory when I was looking for a home. :)